Training24/game.c

```diff
@@ -0,0 +1,38 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------
+// game.c
+// Program on A3 branch.
+// Convert the given decimal number to different forms (Decimal, Hexadecimal, Binary) 
+// ------------------------------------------------------------------------------------------------
+
+#include <malloc.h>
+#include "game.h"
+
+/// <summary>Convert a decimal number to its binary representation.</summary>

```

Comment:
You do not need to give the same comment twice, it is sufficient you give it in the header file. (Unless you are saying something new)